### ISSUE TITLE
Audit mode warning is slightly confusing and incorrect

### ISSUE THREAD

**Issue by erichelgeson** [chef/chef#3255]
Trying out Audit mode and I see this - 
```
$ chef-client -W -r recipe[xyz] -z --config client.rb --audit-mode enabled
[2015-04-21T20:07:33+00:00] WARN: Chef-client has been configured to audit after it converges. Audit mode is an experimental feature currently under development. API changes may occur. Use at your own risk.
* To enable audit mode after converge, use command line option `--audit-mode enabled` or set `:audit_mode = :enabled` in your config file.
* To disable audit mode, use command line option `--audit-mode disabled` or set `:audit_mode = :disabled` in your config file.
* To only run audit mode, use command line option `--audit-mode audit-only` or set `:audit_mode = :audit_only` in your config file.
Audit mode is disabled by default.
Starting Chef Client, version 12.2.1
```

When I read that the first 10 times i took it to mean that --audit-mode was not enabled and I was doing something wrong because it was telling me what options to try. `Also Audit mode is disabled by default.` I took to mean "Even though you asked me to enable it, i disabled it."

The incorrect warning is the config.rb option, putting it as printed in the warning you get 
```
FATAL:  Configuration error SyntaxError: client.rb:4: syntax error, unexpected '=', expecting end-of-input :audit_mode = :enabled
```

it should be `audit_mode :enabled`

**Comment by juliandunn** [comment]
I've proposed that we just [remove it](https://github.com/chef/chef/pulls/3299).

**Comment by erichelgeson** [comment]
Closed via #3299
